### DISCUSSION TITLE
Best way to test messages

### DISCUSSION THREAD

**Discussion by d-cichon** [symfony/symfony#53940] [Q&A]
Hi there!

I'm trying to test Symfony queues functionality, but unfortunately, I can't find many threads on the topic online. It would be great if there was something like an HTTP client, but specifically for queues. Unfortunately, I can't find anything like that. Do you have any advice for me?

**Comment by stloyd** [comment]
I don't know if this would be best but if you want to test the Messenger component, I would highly recommend: [zenstruck/messenger-test](https://github.com/zenstruck/messenger-test)
